posture/serialnumber_macos.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build darwin
```

Comment:
If we'd like this run on any Darwin (macOS, iOS, iPadOS, tvOS) - and I assume we are – you can just rename the file to `serialnumber_darwin.go` and omit the `go:build` tag.

Reply:
I misread the docs, we want to build this on macOS and _not_ iOS, as we are not allowed to get this info. Will update.